classes/bridges/across.js

```diff
@@ -119,7 +101,7 @@ class AcrossBridge extends BaseBridge {
       inputToken: fromToken,
       outputToken: toToken,
     };
-    const quote = await this.getQuoteWithCache(route, amount);
+    const quote = await this.getQuote(route, amount);
     const { feeInUSD, fee } = await this.getFeeCosts(
```

Comment:
would you follow this format in squid.js?
1. first `getQuote()`
2. and then `this.getFeeCosts()`